docs/variants/novacustom_nv4x/building.md

```diff
@@ -30,8 +30,8 @@ $ git clone https://github.com/Dasharo/coreboot.git
 Navigate to the source code directory and checkout to the desired revision:
 
 > Replace the REVISION with one of the:
-> - `clevo_nv41mz/release` for the latest released version
-> - `clevo_nv41mz/vVERSION` (e.g. `v1.0.1`) for the given release
+> - `dasharo_release` for the latest released version
```

Comment:
I don't like this branch name. What board will actually be buildable from this branch? it is nowhere close to be compliant with our rules of branch naming. Was it discussed and approved?

Reply:
Reverted to `novacustom_nv4x/release`

Reply:
@macpijan I almost, Didn't we say we will build all clevo laptops from this branch? Or are we going to document the shared branching first?